Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -938,6 +938,31 @@ theorem lintegral_pos_iff_support {f : α → ℝ≥0∞} (hf : Measurable f) :
   simp [pos_iff_ne_zero, hf, Filter.EventuallyEq, ae_iff, Function.support]
 #align measure_theory.lintegral_pos_iff_support MeasureTheory.lintegral_pos_iff_support
 
+theorem lintegral_positive_iff {C : Set α} (hmC : MeasurableSet C)
+    {h : α → ℝ≥0∞} (hm : AEMeasurable h μ) :
+    (∃ᵐ x ∂μ, x ∈ C ∧ h x ≠ 0) ↔ 0 < ∫⁻ x in C, h x ∂μ := by
```

Comment:
I've just noticed that we already have `lintegral_pos_iff_support`. What about this for a set integral?
```lean
theorem setLintegral_pos_iff {f : α → ℝ≥0∞} (hf : Measurable f) {s : Set α} :
    0 < ∫⁻ a in s, f a ∂μ ↔ 0 < μ (Function.support f ∩ s) := by
  rw [lintegral_pos_iff_support hf, Measure.restrict_apply (measurableSet_support hf)]
```

Reply:
You're right, that's a way better way to prove the same thing, thank you. I commited your changes and removed the `fun_ae_imp_set_ae` lemma.  I will open a new PR for the more general version on Filters.

Reply:
I was just thinking, since this is entirely your work, maybe I should not be the author of this PR, I shall not get credits from your work. I can close this PR or make you a co-author, as you prefer. Anyway, thank you again for your help on this rather chaotic PR.